core/src/test/java/org/apache/brooklyn/core/catalog/internal/SpecParameterInMetaTest.java

```diff
@@ -43,18 +49,40 @@
 public class SpecParameterInMetaTest {
     private ManagementContext mgmt;
     private BrooklynCatalog catalog;
-    private String spec;
+    private String specId;
 
     @BeforeMethod(alwaysRun=true)
     public void setUp() {
         mgmt = LocalManagementContextForTests.newInstanceWithOsgi();
         catalog = mgmt.getCatalog();
-        spec = TestToSpecTransformer.registerSpec(EntitySpec.create(BasicEntity.class));
+        StaticTypePlanTransformer.forceInstall();
+        PlanToSpecFactory.forceAvailable(TestToSpecTransformer.class, JavaCatalogToSpecTransformer.class);
```

Comment:
Intention is to get rid of the `TestToSpecTransformer` but we need a bit more machinery first.  Have added TODO to that effect.